pkg/config/image_test.go

```diff
@@ -29,6 +29,18 @@ func TestParse(t *testing.T) {
 		"beta=bar",
 	}
 	assert.Equal(t, expectedKernelArgs, imageConfig.OperatingSystem.KernelArgs)
+
+	userConfigs := imageConfig.OperatingSystem.Users
+	assert.Len(t, userConfigs, 3)
```

Comment:
Should be `require` so that assertions are guaranteed to only be performed on a slice with the expected length.
```suggestion
	require.Len(t, userConfigs, 3)
```

Reply:
Good pick up